src/MaaWpfGui/Res/Localizations/ja-jp.xaml

```diff
@@ -37,15 +38,21 @@
     <system:String x:Key="OriginStone">製造所-源石の欠片</system:String>
     <system:String x:Key="Chip">製造所-中級SoC</system:String>
     <system:String x:Key="DormTrustEnabled">宿舎の空き位置で信頼を上げる</system:String>
-    <system:String x:Key="DormFilterNotStationedEnabled">作業中のオペレーターを宿舎に移動しない</system:String>
-    <!--  現状動作不明のため対応次第訳修正  -->
-    <system:String x:Key="OriginiumShardAutoReplenishment">源石の欠片は自動で補充する</system:String>
-    <system:String x:Key="CustomInfrastEnabled">カスタムされた基地構成を有効にする（Alpha）</system:String>
+    <system:String x:Key="DormFilterNotStationedEnabled">作業中の人員を宿舎に移動しない（例:訓練所でのアイリーニなど）</system:String>
+    <system:String x:Key="OriginiumShardAutoReplenishment">源石の欠片を自動で補充</system:String>
+    <system:String x:Key="CustomInfrastEnabled">カスタムされた基地構成を有効にする（ベータ）</system:String>
+    <system:String x:Key="DefaultInfrast">標準設定</system:String>
+    <system:String x:Key="UserDefined">ユーザー定義</system:String>
+    <system:String x:Key="153_3">153 3周/日</system:String>
+    <system:String x:Key="243_3">243 3周/日<</system:String>
+    <system:String x:Key="243_4">243 4周/日<y</system:String>
```

Comment:
There is a typo here, there is an extra <y

Reply:
https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/3774/commits/5f8a6ef0b2747e7e96b56e24378d70512d69e250

Thx. fix it.